pandas/core/ops.py

```diff
@@ -788,17 +782,35 @@ def wrapper(self, other, axis=None):
         if axis is not None:
             self._get_axis_number(axis)
 
+        res_name = _get_series_op_result_name(self, other)
+
         if isinstance(other, ABCDataFrame):  # pragma: no cover
             # Defer to DataFrame implementation; fail early
             return NotImplemented
 
+        elif isinstance(other, ABCSeries) and not self._indexed_same(other):
+            raise ValueError('Can only compare identically-labeled Series '
+                             'objects')
```

Comment:
Nit: I prefer not to "orphan" words on their own line.  Move the "Series" part of your sentence there too.

Reply:
Will change.  Ditto comma above.